webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -749,6 +749,16 @@ Kommunicate.richMsgEventHandler = {
         console.log('passenger detail submitted');
     },
     processQuickReplies: function (e) {
+        window.kmWidgetEvents.onRichMessageButtonClick({
```

Comment:
avoid calling 2 things from the main code. call a single thing related to events subsciption from here, and then from the events thing, call both events callback and the google analytics code